lua/yabs/conditions/init.lua

```diff
@@ -1,9 +0,0 @@
-local M = {}
-
-function M.file_exists(file)
-  return function()
-    return require('yabs.utils').file_exists(file)
```

Comment:
Yes, we could do it like this.

> he future we might want to add more included functions as well

What kind of conditions are you planning to add in the future?
I think that the user can add any conditions on his own. Personally, as a user, it would be clearer for me to use some well-known functions that I have already used before.

Reply:
Hmm...I'll have to think about this one. Mainly I had it before because having a file existing was a specifically requested condition case, and the one I found after googling was unintuitive to say the least, so I thought it would be best to include it rather than have the user have to figure it out or add it to the README or something.

That said, that specific case is no longer needed...I'm still undecided about whether we should remove the whole thing because of that. I think we can keep it for now, but I'll certainly think more about it and maybe we can remove it later.

Reply:
Okay, let's call `require('plenary.path'):new(file):exists()` under the hood.
Let me know if you agree to remove it in the future, I can add a deprecation notice via `vim.notify`.

Reply:
Sounds good!

Reply:
Should I call `file_exists` in the next suggestion for now until we deprecate `file_exists`?

Reply:
The `Plenary` option is less elegant, but more straightforward.
The `file_exists` option is shorter, but with it it is less clear what a custom function should look like.

Reply:
Yes, I think that would be best.

Reply:
> The `Plenary` option is less elegant, but more straightforward.
> The `file_exists` option is shorter, but with it it is less clear what a custom function should look like.

That's true...I think we should use the `file_exists` one, but maybe we should make it more clear in the README how the condition keyword should be used as well.

Reply:
Done!